client-api/src/main/java/org/sonarsource/sonarlint/core/clientapi/client/issue/ShowIssueParams.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * SonarLint Core - Client API
+ * Copyright (C) 2016-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarsource.sonarlint.core.clientapi.client.issue;
+
+import java.util.List;
+import org.sonarsource.sonarlint.core.clientapi.common.FlowDto;
+import org.sonarsource.sonarlint.core.clientapi.common.TextRangeDto;
+
+public class ShowIssueParams {
+  private final String fileUri;
+  private final String message;
+  private final String connectionId;
+  private final String ruleKey;
+  private final String issueKey;
+  private final String creationDate;
+  private final String codeSnippet;
+  private final List<FlowDto> flows;
+  private final TextRangeDto textRange;
+
+  public ShowIssueParams(TextRangeDto textRange, String connectionId, String ruleKey, String issueKey, String filePath, String message,
+    String creationDate, String codeSnippet, List<FlowDto> flows) {
```

Comment:
One thing I forgot is that there is no way for clients to know if the issue is a taint or not. We would need that in case we want to select issues in lists

Reply:
Now addressed in [c3a404b](https://github.com/SonarSource/sonarlint-core/pull/722/commits/c3a404b9aa999c1e0c62ec5e54c39ff3c06b5d3b)